docs/guides/tutorials/getting_started/initialize_a_data_context.rst

```diff
@@ -1,77 +1,72 @@
 .. _tutorials__getting_started__initialize_a_data_context:
 
-Set up the tutorial data and initialize a Data Context
+Set up the tutorial and initialize a Data Context
 ===============================================
 
-In Great Expectations, your :ref:`Data Context` manages your project configuration. Using a Data Context is almost always the fastest way to get up and running, even though some teams don't need every component of a Data Context.
+**Welcome to the Great Expectations Getting Started tutorial!** This tutorial will walk you throw a simple exercise where you create an Expectation suite that catches a data issue in a sample data set we provide.
 
+.. admonition:: Prerequisites for the tutorial:
 
-Install Great Expectations
------------------------------------------------
-
-If you haven't already, install Great Expectations.
-
-We recommend deploying within a virtual environment. If you're not familiar with virtual environments, pip, jupyter notebooks,
-or git, you may want to check out the :ref:`supporting_resources` section before continuing.
+  - Make sure you have `Docker <https://www.docker.com/>`_ installed on your machine.
```

Comment:
I think the story of loading to a staging table makes more sense, which is also somewhat required to set up a checkpoint (since we don't necessarily want to teach people to create a checkpoint tied to e.g. the February data file). I totally get your point though - let's chat maybe for the next iteration of this?